crates/katana/rpc/rpc/tests/torii.rs

```diff
@@ -0,0 +1,176 @@
+use std::path::PathBuf;
+use std::sync::Arc;
+use std::time::Duration;
+
+use dojo_test_utils::sequencer::{get_default_test_starknet_config, TestSequencer};
+use jsonrpsee::http_client::HttpClientBuilder;
+use katana_core::sequencer::SequencerConfig;
+use katana_rpc_api::katana::KatanaApiClient;
+use katana_rpc_api::torii::ToriiApiClient;
+use katana_rpc_types::transaction::{TransactionsPage, TransactionsPageCursor};
+use starknet::accounts::{Account, Call};
+use starknet::core::types::FieldElement;
+use starknet::core::utils::get_selector_from_name;
+use tokio::time::sleep;
+
+use crate::common::prepare_contract_declaration_params;
+
+mod common;
+
+pub const ENOUGH_GAS: &str = "0x100000000000000000";
+
+#[tokio::test(flavor = "multi_thread")]
+async fn test_get_transactions() {
```

Comment:
can we add another variant of this test for `instant` mining case ?

Reply:
yes sir